### PR TITLE
fix OAuthSimple require bug :trollface:

### PR DESCRIPTION
**Pull request by denzelwamburu**
Fix for the Issue https://github.com/aksalj/pesapaljs/issues/10.

### CODE REVIEW

File: lib/internal/core.js [hunk 1]
```diff
@@ -10,7 +10,7 @@
  *  Description :
  *
  */
-var OAuthSimple = require('oauthsimple');
+var OAuthSimple = require('../../node_modules/oauthsimple/js/OAuthSimple.js');
```
Comment (aksalj) [inline lib/internal/core.js]: :-1: this could get real messy if this file (`lib/internal/core.js`) ever moves...

File: example/web/package.json [hunk 2]
```diff
@@ -1,14 +1,15 @@
...
+  "scripts": {
+    "start": "node app.js"
+  },
+  "dependencies": {
+    "body-parser": "x.x.x",
+    "express": "4.x.x",
+    "jade": "x.x.x",
+    "lorem-ipsum": "x.x.x",
+    "morgan": "1.5.x",
+    "oauthsimple": "0.0.1"
```
Comment (aksalj) [inline example/web/package.json]: `oauthsimple` is already a dependency of the library. if `pesapaljs`'s dependencies are installed, we don't need this.

**Comment by aksalj** [comment]
:-1:

**Comment by denzelwamburu** [comment]
Yeah that's right, didn't pay much attention there.

**Pull request closed without merging**